rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -2271,7 +2355,11 @@ public ResponseEntity<?> patchProjectWithNetwork(
             log.error(sw360Exception.getWhy());
             return ResponseEntity.badRequest().body(sw360Exception.getWhy());
         }
-
+        String comment = (String) reqBodyMap.get("comment");
+        if (!restControllerHelper.isWriteActionAllowed(updateProject, user) && comment == null) {
+            return new ResponseEntity<>(RESPONSE_BODY_FOR_MODERATION_REQUEST_WITH_COMMIT, HttpStatus.ACCEPTED);
```

Comment:
In case of missing request parameter, shouldn't the response status be `BAD_REQUEST`?